instances/technique/electrocardiography.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/electrocardiography",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "Electrocardiography is the process of producing an electrogram of the heart: a graph of voltage versus time of the electrical activity of the heart using electrodes placed on the skin.",
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0103701",
```

Comment:
this is not the same term. One is referring to the technique, the other to the protocol. Please set this one to null. We will integrate the term in the technique ontology and update your entry when an ID is available. @tgbugs FYI